cirq/ops/common_gates_test.py

```diff
@@ -622,6 +622,8 @@ def test_iswap_str():
 def test_iswap_unitary():
     cirq.testing.assert_allclose_up_to_global_phase(
         cirq.unitary(cirq.ISWAP),
+        # Reference for the iswap gate's matrix using +i instead of -i:
+        # https://quantumcomputing.stackexchange.com/questions/2594/
```

Comment:
Link to answer: https://quantumcomputing.stackexchange.com/a/2596/362

Reply:
I purposefully linked to the question instead of the answer. Makes the link shorter and is more future-proof.